src/sysext/sysext.c

```diff
@@ -533,15 +533,18 @@ static int merge_subprocess(Hashmap *images, const char *workspace) {
                                                "Extension image contains /usr/lib/os-release file, which is not allowed (it may carry /etc/os-release), refusing.");
 
                 /* Now that we can look into the extension image, let's see of the OS version is compatible */
-                r = parse_os_release(
+                r = parse_extension_release(
                                 p,
+                                img->name,
                                 "ID", &extension_os_release_id,
                                 "VERSION_ID", &extension_os_release_version_id,
                                 "SYSEXT_LEVEL", &extension_os_release_sysext_level,
```

Comment:
these three variables should be renamed i guess. extension_os_release_xyz → extension_release_xyz